File: 프로그래머스/0/120909. 제곱수 판별하기/제곱수 판별하기.java
import java.util.*;

class Solution {
    public int solution(int n) {
        int answer = 0;
        for(int i=0;i<n;i++){
            answer = Math.sqrt(n)%1 == 0 ? 1 : 2;
        }
        return answer;
    }
}

File: 프로그래머스/0/120909. 제곱수 판별하기/제곱수 판별하기.c
#include <stdio.h>
#include <stdbool.h>
#include <stdlib.h>
#include <math.h>

int solution(int n) {
    int answer = 0;
    for(int i=0;i<n;i++){
        if(i*i == n){
            return 1;
        }else if(i==n-1){
            return 2;
        }
    }
}